### PR TITLE
added designer news extension

### PR DESCRIPTION
**Pull request by itsnwa**
## Description

New Designer News extensions that display the latest stories from designernews.co

## Screencast

CleanShot 2022-02-19 at 00 02 55@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

**Comment by mathieudutour** [comment]
because of https://github.com/raycast/extensions/pull/941/files#diff-8ccdcc9a95fec0a0750bfd916c49083ac7fa27cc6fad8b6b6dc8b934f2e5a917R18. If you want to publish it, it should use the public version (eg. `1.28.0`)

File: extensions/designer-news/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "name": "designer-news",
+  "title": "Designer News",
+  "description": "See top design stories",
+  "icon": "command-icon.png",
+  "author": "itsnwa",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Designer News",
```
Comment (mathieudutour) [inline extensions/designer-news/package.json:12]: Our guidelines are to have the verb for the title and the service as the subtitle

```suggestion
      "title": "Read Top News",
      "subtitle": "Designer News",
```

Comment (itsnwa) [reply extensions/designer-news/package.json]: You're right @mathieudutour, and yes, it's top news :)

**Review by mathieudutour — APPROVED** [review]
Awesome, let's get this published 🎉

**Pull request merged**